### PR TITLE
Intercept logout event + better error handling

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**

### CODE REVIEW

**Review by sebbousquet — APPROVED** [review]

File: projects/arlas-toolkit/assets/i18n/en.json [hunk 1]
```diff
@@ -231,14 +233,25 @@
...
+	"go to arlas hub": "Go to ARLAS-Hub.",
+	"The connection to ARLAS backend services is lost": "The connection to ARLAS backend services is lost.",
+	"The connection to ARLAS-server is lost": "The connection to ARLAS-server is lost.",
+	"The connection to ARLAS-persistence is lost": "The connection to ARLAS-persistence is lost.",
 	"The connection to dashbords is lost": "The connection to the dashbords is lost",
+	"An error occured in ARLAS backend services": "An error occured in ARLAS backend services.",
+	"An error occured in ARLAS-server": "An error occured in ARLAS-server.",
+	"An error occured in ARLAS-persistence": "An error occured in ARLAS-persistence.",
+	"ARLAS Front-end Settings error": "ARLAS Front-end Settings error",
+	"Please check if the settings.yaml file is provided and has a valid syntax": "Vérifier que le fichier settings.yaml est fourni et que sa syntaxe est valide.",
```
Comment (QuCMGisaia) [inline projects/arlas-toolkit/assets/i18n/en.json:249]: Should be in english

File: projects/arlas-toolkit/assets/i18n/fr.json [hunk 2]
```diff
@@ -241,13 +241,24 @@
...
+  "no": "non",
+  "access not authorized": "Vous n'êtes pas autorisé(e) à accéder à cette page.",
+  "acces forbidden": "Vous n'êtes pas autorisé(e) à accéder à cette page.",
+  "ARLAS ecountered an error": "ARLAS a rencontré une erreur",
+  "The connection to ARLAS backend services is lost": "La connexion aux backends ARLAS a échoué.",
+	"The connection to ARLAS-server is lost": "La connexion à ARLAS-server a échoué.",
+	"The connection to ARLAS-persistence is lost": "La connexion à ARLAS-persistence a échoué.",
+  "An error occured in ARLAS backend services": "Une erreur s'est produite dans les services backend d'ARLAS.",
+	"An error occured in ARLAS-server": "Une erreur s'est produite au niveau d'ARLAS-server.",
+	"An error occured in ARLAS-persistence": "Une erreur s'est produite dans au niveau d'ARLAS-persistence.",
```
Comment (QuCMGisaia) [inline projects/arlas-toolkit/assets/i18n/fr.json:254]: Unnecessary dans

File: projects/arlas-toolkit/assets/i18n/fr.json [hunk 3]
```diff
@@ -241,13 +241,24 @@
...
+  "ARLAS Front-end Settings error": "Erreur de paramétrage de l'interface ARLAS",
+  "Please check if the settings.yaml file is provided and has a valid syntax": "Please check if the settings.yaml file is provided and has a valid syntax.",
```
Comment (QuCMGisaia) [inline projects/arlas-toolkit/assets/i18n/fr.json:256]: Should be in french

**Review by QuCMGisaia — APPROVED** [review]

File: projects/arlas-toolkit/assets/i18n/fr.json (hunk 2 above)
Comment (MohamedHamouGisaia) [reply projects/arlas-toolkit/assets/i18n/fr.json]: fixed

File: projects/arlas-toolkit/assets/i18n/fr.json (hunk 3 above)
Comment (MohamedHamouGisaia) [reply projects/arlas-toolkit/assets/i18n/fr.json]: fixed

File: projects/arlas-toolkit/assets/i18n/en.json (hunk 1 above)
Comment (MohamedHamouGisaia) [reply projects/arlas-toolkit/assets/i18n/en.json]: fixed

**Pull request merged**